api/db/migrations/20241206100509_create-legal-document-versions-user-acceptances.js

```diff
@@ -0,0 +1,27 @@
+const TABLE_NAME = 'legal-document-versions-user-acceptances';
```

Comment:
Idéalement pour être cohérent avec le nommage des autres tables : 
```suggestion
const TABLE_NAME = 'legal-document-version-user-acceptances';
```